sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/AnomalyDetectorModelFactory.cs

```diff
@@ -87,9 +87,19 @@ public static ModelInfo ModelInfo(int? slidingWindow = null, AlignPolicy alignPo
         /// <summary> Initializes a new instance of ErrorResponse. </summary>
         /// <param name="code"> The error code. </param>
         /// <param name="message"> The message explaining the error reported by the service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="code"/> or <paramref name="message"/> is null. </exception>
         /// <returns> A new <see cref="Models.ErrorResponse"/> instance for mocking. </returns>
         public static ErrorResponse ErrorResponse(string code = null, string message = null)
         {
+            if (code == null)
```

Comment:
You should use the `Argument` helper class in Azure.Core shared source: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Shared/Argument.cs

Reply:
Do you mean the generator should do this? This is generated code.

Reply:
I may be biased since I wrote `Argument`, but I used to think the same thing: the generator should use it. However, it was created mostly for dev productivity and as it's grown over the years, think it's fine if the generator does what we see here now. If a project doesn't otherwise link _Argument.cs_ why impact the assembly size unnecessarily? Just my $0.02.

Reply:
I would tend to agree with you - we don't really need to worry about the productivity of the generator 😄